packages/app/src/components/SearchBar/index.js

```diff
@@ -16,16 +16,19 @@ const SearchBar = () => {
 
 
   function findSearchResults(searchTerm) {
-    const filteredNoms = NominationsData.filter((nomination) => {
-      return [
-        formatSearch(nomination.providerName),
-        formatSearch(nomination.patientName),
-        formatSearch(nomination.nominationName),
-        formatSearch(nomination.representativeName),
-      ].some((nom) => nom.includes(searchTerm));
-    });
-    setSearchResultData(filteredNoms);
-
+    let filteredNoms = []
+    console.log('NominationsData ', NominationsData)
```

Comment:
Please remove console.log as I assume it was used for debugging. If it was used for debugging I would recommend using `debugger` as it will actually stop processing in your browser and let you look explore in chrome. Let me know if you would like me to show you this.